rickshaw-settings.json

```diff
@@ -9,7 +9,7 @@
 	}
     },
     "userenvs": {
-	"default": "rhubi8"
+	"default": "fedora39"
```

Comment:
Are we sure we want to tie the default userenv for benchmarks to the default userenv for tools?